Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -276,6 +404,17 @@ DECLARE_SOA_TABLE(UnassignedTracks, "AOD", "UNASSIGNEDTRACK",
 
 using UnassignedTrack = UnassignedTracks::iterator;
 
+namespace unassignedfwdtracks
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+DECLARE_SOA_INDEX_COLUMN(FwdTrack, fwdtrack);
```

Comment:
I think this is for MFTTracks and should point to MFTTrack
For the FwdTrack, we can solve the unassigned ones through the BC Index, so we don't need a table for them

Reply:
Aciually I think we need unnasigned tracks for both FwdTracks and MFTTracks. FwdTracks will be assigned to BC index when there is MID match. Which is not true for `MCHStandalone` and `GlobalForwardTrack`.

Reply:
GlobalForwardTrack has a link to MFT and through that is assigned (or in the unassigned table)
For MCHStandalone @pillot said you would estimate the most likely BC even if there is some uncertainty if I remember the discussion well...

Reply:
You are right.